doc/en/user/source/services/wfs/outputformats.rst

```diff
@@ -135,3 +135,15 @@ JSON output ``format_options``:
 JSON output ``system properties``:
 
 * ``json.maxDepth=<max_value>`` is used to determine the max number of allowed JSON nested objects on encoding phase.  By default the value is 100.
+
+
+CSV output
+----------------
+
+A Default CSV file uses a comma to separate values. Each line of the file is a data record. Each record consists of one or more fields, separated by commas. The separator can be changed using format_options as specified below.
+
+csv file output ``format_options``:
+
+* ``format_option=filename:<file>``: if a file name is provided, the name is used as the output file name. For example, ``format_options=filename:roads.zip``.
```

Comment:
even though correct, I find it a bit counter-intuitive to have a CSV file with `.zip` extension
```suggestion
* ``format_option=filename:<file>``: if a file name is provided, the name is used as the output file name. For example, ``format_options=filename:roads.csv``.
```

Reply:
Changes applied, Thank you :)